test/data/v3/with-k8s-fake-client/templates/lookup.yaml

```diff
@@ -0,0 +1,3 @@
+---
+lookup_exists: {{ (lookup "v1" "Pod" "default" "exists") | toYaml | nindent 2 }}
+lookup_not_exists: {{ (lookup "v1" "Pod" "default" "not-exists") | toYaml | nindent 2 }}
```

Comment:
new line is missing

Reply:
Could this help with testing
```
{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
apiVersion: policy/v1
{{- else }}
apiVersion: policy/v1beta1
{{- end }}
```

If yes, how example test is going to looks like?

Reply:
No, this feature is only for the lookup function.

But you can already write tests to test against the apiVersion.
```
capabilities:
  apiVersions:
    - policy/v1
```